docs/markdown/authoritative/backend-generic-sql.md

```diff
@@ -0,0 +1,211 @@
+# Generic SQL Backends
+The generic SQL backends (like gmysql, gpgsql and godbc) are backends with easily
+configurable SQL statements, allowing you to graft PowerDNS on any SQL database
+of your choosing. Because all database schemas will be different, a generic
+backend is needed to cover all needs.
+
+**Warning**: Host names and the MNAME of a SOA records are NEVER terminated with
+a '.' in PowerDNS storage! If a trailing '.' is present it will inevitably cause
+problems, problems that may be hard to debug.
+
+# Basic functionality
+4 queries are needed for regular lookups and 1 is needed for zone transfers.
+
+The 4+4 regular queries must return the following 8 fields, in this exact order:
```

Comment:
you say `4 queries for regular... 1 for zone transfers` above and suddenly they are 4+4?